db/migrate/20170719210832_move_versions_table.rb

```diff
@@ -0,0 +1,5 @@
+class MoveVersionsTable < ActiveRecord::Migration
+  def change
+    rename_table :versions, :vestal_versions
```

Comment:
Per the gem source, oracle does rename the table and indexes. (https://github.com/rsim/oracle-enhanced/blob/c2caf15919e3d34129287958be4f2301a595b972/lib/active_record/connection_adapters/oracle_enhanced/schema_statements.rb). 

The other... I guess we need to test, as far as I can see it should work.